src/main/java/gameframework/drawing/InputManager.java

```diff
@@ -0,0 +1,51 @@
+package gameframework.drawing;
+
+import java.awt.Canvas;
+import java.awt.event.KeyListener;
+import java.awt.event.MouseListener;
+
+/**
+ * Manager that handles all the inputs which must have an effect on a given canvas.
+ */
+public class InputManager {
+	
+	/**
+	 * The canvas whose inputs are managed by this manager
+	 */
+	private final Canvas canvas;
```

Comment:
please use `protected` instead. You never know what your subclass or test class will need.